src/nyc_trees/nyc_trees/settings/production.py

```diff
@@ -82,3 +82,20 @@ def get_env_setting(setting):
     logger.exception('Failed to resolve TILER_HOST, %s' %
                      environ.get('TILER_HOST'))
 # END TILER CONFIGURATION
+
+# Django Storages CONFIGURATION
+mac_metadata = instance_metadata['network']['interfaces']['macs']
+vpc_id = mac_metadata.values()[0]['vpc-id']
+
+# The VPC id should stay the same for all app servers in a particular
+# environment and remain the same after a new deploy, but differ between
+# environments.  This makes it a suitable S3 bucket name
+AWS_STORAGE_BUCKET_NAME = 'django-storages-{}'.format(vpc_id)
+
```

Comment:
Setting `AWS_QUERYSTRING_AUTH` to `False` should remove all of the additional query string parameters, but [doesn't appear to work](https://github.com/boto/boto/issues/1477) when a security token is used to establish the S3 connection. Workarounds exist, but not sure they're worth chasing.